specs/standard-reports.spec

```diff
@@ -0,0 +1,212 @@
+* The User is currently viewing 'step 2 of 7'
+* select 'Postcode district (e.g. ‘GL1’)'
+* Click button to advance to 'Next' screen
+* The User is currently viewing 'step 3 of 7'
+* The User can input text
+* Input "BS49" to the input field
+* The User is currently viewing 'step 4 of 7'
+* The User can see 2 options for location specification
+* Select 'Don't aggregate, just show the total'
+* Click button to advance to 'Next' screen
+* Advance to step 5
+
+## Geographical - postcode sector
+
+Tags: standard, report, location
+
+* Repeat step 1
+* The User is currently viewing 'step 2 of 7'
+* select 'Postcode sector (e.g. ‘GL1 2’)'
+* Click button to advance to 'Next' screen
+* The User is currently viewing 'step 3 of 7'
+* The User can input text
+* Input "BS49" to the input field
+* The User is currently viewing 'step 4 of 7'
+* The only option is prefilled for 'Don't aggregate, just show the total'
+* Click button to advance to 'Next' screen
+* Advance to step 5
+
+## time period - step 5
+
+Tags: standard, report, time
+
+* The User is currently viewing 'step 5 of 7'
+* The User can select a number of time periods to generate a report from
+* Select 'Year to date'
+* Click button to advance to 'Next' screen
+
+## Visible dates - step 5
+
+Tags: standard, report, time
+
+* Repeat step 1
+* Repeat Geographical - country - step 2
+* The User is currently viewing 'step 5 of 7'
+* The User can see the current year and select 'current year to date'
+* The User can only select months in the current year up until the 'previous month'
+* The User can see the previous year and select 'previous year all year'
+* The User can click more dates to expand a greater selection of options
+* The User can see all years dating back to 1995
+* The User can select any month within a full year
+
+## additional options - step 6
+
+Tags: standard, report
+
+* The User is currently viewing 'step 6 of 7'
+* The User can select different property types
+* A User cannot progress until a selection is made
+* Select 'Both old and new properties'
+
+## summary display - step 7
+
+Tags: standard, report
+
+* The User is currently viewing 'step 7 of 7'
+* The User can read the parameters they have chosen in previous steps
+* Select "change..." adjacent 'report type is average prices and volumes'
+* The User is routed to step 1
+* Select 'Banded prices'
+* Click button to advance to 'Next' screen
+* The User is routed back to step 7
+
+## report generation
+
+Tags: standard, report
+
+* Repeat step 1
+* Repeat Geographical - country - step 2
+* The User is currently viewing 'step 5 of 7'
+* Select 'Year to date'
+* Select 'Latest quarter for which data is available'
+* Select 'Latest month for which data is available'
+* Select '2021 to date'
+* Select '2020 all year'
+* Click button to advance to 'Next' screen
+* Repeat step 6
+* Click 'Generate report' button
+* Whilst generating the User will see 'in progress, estimated complete in' with a given timeframe.
```

Comment:
There's a nuance here. If you select a set of parameters for which a standard report has been previously generated, the result will be served from S3 cache. So instead of  position in the queue or "in progress", you will go straight to the download links.

These cached reports are cleared once per month when the source data is updated.